.github/workflows/macos-amd64-build.yml

```diff
@@ -9,11 +9,15 @@ jobs:
     - uses: actions/checkout@v2
       with:
         submodules: recursive
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+    - name: remove system python
+      run: |
+        sudo rm -rf /Library/Frameworks/Python.framework/Versions/3* /usr/local/bin/python3* || true
```

Comment:
On my Mac, when I tried to installed a new version of python, I was told not simply replace the directory. Let's see what will happen.